settings.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+	// Gets URL of homepage
+	function homePageURL() {
```

Comment:
>homePageURL [](start = 10, length = 11)

Functions that are not part of a namespaced class need to be prepended with a slug identifying this plugin.